src/typings/tacoLiveReload.d.ts

```diff
@@ -53,6 +53,7 @@ declare module TacoLiveReload {
     export interface LiveReloadOptions {
         ignore: string,
         ghostMode: boolean,
+        tunnel: boolean,
```

Comment:
Are these options exposed in commands.json? In other words, shouldn't we enumerate these "new" options are supported options for "run" and "emulate" on "taco help"?